keyboards/kbd0/curve0/60_ansi/keymaps/default/keymap.json

```diff
@@ -0,0 +1,22 @@
+{
+    "author": "kbd0",
+    "keyboard": "kbd0/curve0/60_ansi",
+    "keymap": "default",
+    "layout": "LAYOUT",
+    "layers": [
+        [
+          "KC_ESC", "KC_1", "KC_2", "KC_3", "KC_4", "KC_5", "KC_6", "KC_7", "KC_8", "KC_9", "KC_0", "KC_MINS", "KC_EQL", "KC_BSLS", "KC_BSPC",
+          "KC_TAB", "KC_Q", "KC_W", "KC_E", "KC_R", "KC_T", "KC_Y", "KC_U", "KC_I", "KC_O", "KC_P", "KC_LBRC", "KC_RBRC", "KC_BSLS",
+          "KC_CAPS", "KC_A", "KC_S", "KC_D", "KC_F", "KC_G", "KC_H", "KC_J", "KC_K", "KC_L", "KC_SCLN", "KC_QUOT", "KC_ENT",
+          "KC_LSFT", "KC_NUBS", "KC_Z", "KC_X", "KC_C", "KC_V", "KC_B", "KC_N", "KC_M", "KC_COMM", "KC_DOT", "KC_SLSH", "KC_RSFT",
+          "MO(1)", "KC_LCTL", "KC_LGUI", "KC_LALT", "KC_SPC", "KC_RALT", "KC_RGUI", "KC_APP", "KC_RCTL"
+        ],
+        [
+          "KC_GRV", "KC_F1", "KC_F2", "KC_F3", "KC_F4", "KC_F5", "KC_F6", "KC_F7", "KC_F8", "KC_F9", "KC_F10", "KC_F11", "KC_F12", "_______", "KC_DEL",
+          "_______", "_______", "KC_UP", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",
+          "_______", "KC_LEFT", "KC_DOWN", "KC_RGHT", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",
+          "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",
+          "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______"
```

Comment:
```suggestion
          "KC_ESC",  "KC_1",    "KC_2",    "KC_3",    "KC_4",    "KC_5",    "KC_6",    "KC_7",    "KC_8",    "KC_9",    "KC_0",    "KC_MINS", "KC_EQL",  "KC_BSLS", "KC_BSPC",
          "KC_TAB",  "KC_Q",    "KC_W",    "KC_E",    "KC_R",    "KC_T",    "KC_Y",    "KC_U",    "KC_I",    "KC_O",    "KC_P",    "KC_LBRC", "KC_RBRC", "KC_BSLS",
          "KC_CAPS", "KC_A",    "KC_S",    "KC_D",    "KC_F",    "KC_G",    "KC_H",    "KC_J",    "KC_K",    "KC_L",    "KC_SCLN", "KC_QUOT",            "KC_ENT",
          "KC_LSFT", "KC_NUBS", "KC_Z",    "KC_X",    "KC_C",    "KC_V",    "KC_B",    "KC_N",    "KC_M",    "KC_COMM", "KC_DOT",  "KC_SLSH", "KC_RSFT", "MO(1)",
          "KC_LCTL", "KC_LGUI", "KC_LALT",                                  "KC_SPC",                                    "KC_RALT", "KC_RGUI", "KC_APP", "KC_RCTL"
        ],
        [
          "KC_GRV", "KC_F1",    "KC_F2",   "KC_F3",   "KC_F4",   "KC_F5",   "KC_F6",   "KC_F7",   "KC_F8",   "KC_F9",   "KC_F10",  "KC_F11",  "KC_F12",  "_______", "KC_DEL",
          "_______", "_______", "KC_UP",   "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",
          "_______", "KC_LEFT", "KC_DOWN", "KC_RGHT", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",            "_______",
          "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______", "_______",
          "_______", "_______", "_______",                                  "_______",                                  "_______", "_______", "_______", "_______"
```

Reply:
I also need to add keymaps/default_ansi_60 and keymaps/default_60_ansi_split_bs_rshift , correct?

After replacing LAYOUT with LAYOUT_ALL I can no longer build the firmware

make kbd0/curve0/60_ansi:default

```
Making kbd0/curve0/60_ansi with keymap default

☒ Invalid JSON keymap: C:/.../qmk_firmware/keyboards/kbd0/curve0/60_ansi/keymaps/default/keymap.json : 'LAYOUT_ALL' is not valid under any of the given schemas
```

Reply:
You do not need to add additional keymaps for all the possible layouts, it's fine just for them to exist in `keyboard.json`.

You received this error because the layout naming syntax expects `LAYOUT_xxx` where `xxx` is lowercase. So it should be `LAYOUT_all`.

Reply:
Wow haha had that wrong for the longest time

Without the additional keymaps, what happens if a user chooses that layout in configurator? Do they need to fill in the entire layout themselves or does it infer from LAYOUT_all

Reply:
> Without the additional keymaps, what happens if a user chooses that layout in configurator? Do they need to fill in the entire layout themselves or does it infer from LAYOUT_all

The former.